pxtblocks/fields/field_sound_effect.ts

```diff
@@ -13,6 +13,7 @@ namespace pxtblockly {
         waveFieldName: string;
         interpolationFieldName: string;
         effectFieldName: string;
+        useMixerSynthesizer: any;
```

Comment:
from the looks of it, this is set as any because the param is usually a string but after the constructor you scope this down to boolean with isTrue? Almost feels worth adding an extend interface to avoid `any` here to me like this

```typescript
        export interface ParsedFieldSoundEffectOptions extends FieldSoundEffectParams {
            useMixerSynthesizer: boolean;
        }
```

but that's pretty nitpicky